spec/config_spec.js

```diff
@@ -0,0 +1,36 @@
+const setConfig = require('../src/config.js')
+
+describe('setting up config object', function() {
+  it('can accept 0 arguments and returns default config', function() {
+    expect(setConfig()).toEqual({
+      host: 'metrics-api.iopipe.com',
+      path: '/v0/event',
+      clientId: '',
+      debug: false,
+      networkTimeout: 5000,
+      captureTimeouts: true,
+      timeoutWindow: 150,
```

Comment:
@pselle Could we axe `captureTimeouts` var and just use `timeoutWindow` and update our lang to say "set to 0" if you want to opt out?